scripts/cli/tests/1_poly_transfer.js

```diff
@@ -4,37 +4,40 @@ module.exports = require("../util/init.js");
 
 let { reqImports } = require("../util/init.js");
 
+// Sets the default exit code to fail unless the script runs successfully
+process.exitCode = 1;
+
 async function main() {
   // Schema path
-  const filePath = reqImports["path"].join(__dirname + "/../../../polymesh_schema.json");
-  const customTypes = JSON.parse(reqImports["fs"].readFileSync(filePath, "utf8"));
+  const filePath = reqImports.path.join(__dirname + "/../../../polymesh_schema.json");
+  const customTypes = JSON.parse(reqImports.fs.readFileSync(filePath, "utf8"));
 
   // Start node instance
-  const ws_provider = new reqImports["WsProvider"]("ws://127.0.0.1:9944/");
-  const api = await reqImports["ApiPromise"].create({
+  const ws_provider = new reqImports.WsProvider("ws://127.0.0.1:9944/");
+  const api = await reqImports.ApiPromise.create({
```

Comment:
`ApiPromise` function could do internally the 3 steps above:
1. Create the filePath
2. Load custom types.
3. Create the ws_provider.
4. Return `api`

Reply:
Makes sense, I'll start working on it

Reply:
finished it